modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/utilities/forms/DDMFormHandler.js

```diff
@@ -47,15 +47,19 @@ class DDMFormHandler {
 			body: fieldsParam,
 			headers: new Headers({'x-csrf-token': Liferay.authToken}),
 		}).then((cpInstance) => {
-			if (cpInstance.cpInstanceExist) {
-				const dispatchedPayload = {
-					addToCartId: this.addToCartId,
-					cpInstance,
-					formFields: this.fields,
-				};
+			var dispatchedPayload = {
```

Comment:
`const` can work here, it's an `object` whose keys are not altering the reference when re-assigned. :)